civihr_default_theme/assets/sass/partials/_panel.scss

```diff
@@ -18,6 +14,10 @@
   max-height: 162px;
 }
 
+.panel-body-wrap--collapse {
```

Comment:
Why not:
`.panel-body-wrap {
  transition: max-height .4s linear;
  max-height: 162px;
  
  &--collapse {
    max-height: 0;
  }
}` ?